Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml

```
@@ -0,0 +1,180 @@
+          [bold]Shift Time (upon arrival):[/bold] 
+
+      [head=3]Sustained Injuries[/head]
+          [bold]Brute Injuries:[/bold]
+          \[  ] Blunt     \[  ] Slash     \[  ] Piercing
+
+          [bold]Burn Injuries:[/bold]
+          \[  ] Heat     \[  ] Shock     \[  ] Cold     \[  ] Caustic
+
+          [bold]Toxin Injuries:[/bold]
+          \[  ] Poison     \[  ] Radiation
+
+          [bold]Asphyxiation Injuries:[/bold]
+          \[  ] Bloodloss     \[  ] Airloss
+
+      [head=3]Administered Treatment[/head]
+          [bold]Applied Topicals:[/bold]
+          \[  ] Bruise pack     \[  ] Ointment     \[  ] Gauze     \[  ] Blood pack
+          \[  ] Sutures     \[  ] Regenerative mesh
+
+          [bold]Injected Chemicals:[/bold]
+          [bullet/] Chemical, dosage: XX u
+          [bullet/] Chemical, dosage: XX u
+
+
+
+      [color=#AAAAAA]This Form is void without the signature and stamp of a medical specialist[/color]
+
+- type: entity
+  id: PaperWrittenMedicalFormPrescription
+  name: Medical Prescription
+  suffix: MappedPaper
+  parent: Paper
+  components:
+  - type: Paper
+    content: |2
+      [color=#52B4E9]◥[bold]N[/bold]◣ [bold]Medical Contractor Standards[/bold][/color]
+      [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+      [head=2]Medical Prescription[/head]
+
+      [head=3]Patient Information[/head]
+          [bold]Name:[/bold] 
+          [bold]Species:[/bold] 
+          [bold]Gender:[/bold] 
+          [bold]Position Title:[/bold] 
+          [bold]Vessel IFF:[/bold] 
+
+      [head=3]Prescription[/head]
+          [bullet/] CHEMICAL, dosage: XX u
+          [bullet/] CHEMICAL, dosage: XX u
+
+          [bold]WRITE ANY INSTRUCTIONS HERE.[/bold]
+
+
+
+      [color=#AAAAAA]This Form is void without the signature and stamp of a medical specialist[/color]
+
+- type: entity # Based on form made by Radiantflash (Lykaon Rahmer)
+  id: PaperWrittenMedicalFormHealthInsurance
+  name: Health Insurance Contract
+  suffix: MappedPaper
+  parent: Paper
+  components:
+  - type: Paper
+    content: |2
+      [color=#52B4E9]◥[bold]N[/bold]◣ [bold]Medical Contractor Standards[/bold][/color]
+      [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+      [head=2]Health Insurance Contract[/head]
+
+      [head=3]Policy Holder Information[/head]
+          [bold]Name:[/bold] 
+          [bold]Species:[/bold] 
+          [bold]Gender:[/bold] 
+          [bold]Occupation:[/bold] 
+          [bold]Vessel IFF:[/bold] 
+
+      [head=3]Plan Coverage[/head]
+          [bold]Medical Coverage:[/bold]
+          \[  ] Treatment: Coverage up to $5 000. ($2 000 premium)
+          \[  ] Revival: Coverage up to $10 000. ($2 000 premium)
+          \[  ] Cloning Fee: Coverage up to $25 000. ($5 000 premium)
+
+          [bold]Additional Coverage:[/bold]
+          \[  ] Towing: Coverage up to $10 000. Small shuttles only. ($2 000 premium)
+
+      [head=3]Grand Total[/head]
+          [bold]$XXX XXX[/bold]
```

Might be annoying, but I think saying this is the total of the premium vs. the coverage (which could also be listed, but I don't think it's critical)
```suggestion
      [head=3]Total Premium[/head]
          [bold]$XX XXX[/bold]
```